CMakeLists.txt

```
@@ -139,11 +108,63 @@ set(CLP_FFI_JS_SRC_ZSTD
     src/submodules/zstd/lib/decompress/zstd_decompress.c
 )
 
-target_sources(
-    ${CLP_FFI_JS_BIN_NAME}
-    PRIVATE
+set(CLP_FFI_JS_SUPPORTED_ENVIRONMENTS
+    "node;worker"
```

```suggestion
    node
    worker
```

I prefer separating them on different lines but feel free to disagree.